src/main/java/mekanism/common/block/BlockTransmitter.java

```diff
@@ -231,49 +230,54 @@ public boolean onBlockActivated(World world, BlockPos pos, IBlockState state, En
 
 		return false;
 	}
-	
-	@Override
-    public ItemStack getPickBlock(IBlockState state, RayTraceResult target, World world, BlockPos pos, EntityPlayer player)
+
+	private void dismantleBlock(IBlockState state, World world, BlockPos pos)
 	{
-		ItemStack itemStack = ItemStack.EMPTY;
+		dropBlockAsItem(world, pos, state, 0);
+		world.setBlockToAir(pos);
+	}
+
+	private ItemStack getDropItem(IBlockAccess world, BlockPos pos) {
 		TileEntitySidedPipe tileEntity = getTileEntitySidedPipe(world, pos);
 		if(tileEntity != null)
 		{
-			itemStack = new ItemStack(MekanismBlocks.Transmitter, 1, tileEntity.getTransmitterType().ordinal());
-			
-			if(!itemStack.hasTagCompound())
```

Comment:
This will wipe any data a capability may have added.

Reply:
The constructor ```ItemStack(Block, int, int)``` (calls ```ItemStack(Item, int, int)```) is being used, so the ```NBTTagCompound``` should always be null.

Reply:
It does not matter what constructor you use.

Reply:
I must have overlooked ```forgeInit```